tests/data/model-validation.ts

```diff
@@ -779,15 +779,12 @@ type org
 `,
     expectedError: [
       {
-        endColumn: 23,
+        endColumn: 22,
```

Comment:
These seem to have broken?

Reply:
Previously the grammar itself allows empty direct assignable type and we rely on the checkDSL script to see whether there are any types.  However, with the grammar update, the grammar itself now prevents user from specifying empty assignable types.  This makes the check cleaner.